cmake/noexcep_ops.cmake

```diff
@@ -10,11 +10,11 @@ if (OCOS_ENABLE_WORDPIECE_TOKENIZER)
     message(FATAL_ERROR "WORDPIECE_TOKENIZER operator needs c++ exceptions support")
 endif()
 if (OCOS_ENABLE_BLINGFIRE)
-    message(STATUS "BLINGFIRE operator needs c++ exceptions support, enable exceptions by default!")
+    message(FATAL_ERROR "BLINGFIRE operator needs c++ exceptions support")
```

Comment:
@wenbingl this looked like a mistake to me since for all other cases it's an error message. let me know if that's not the case.

Reply:
Good  catch!

Reply:
This was set to STATUS in #143 based on blingfire completely resetting the CXX_FLAGS, so the flags to disable exceptions were essentially ignored. 

That doesn't seem like a good setup because if you build with exceptions disabled in ort-ext and enabled in blingfire, if blingfire throws it won't be handled. 

Based on that I think setting it to FATAL_ERROR is more correct